ElectronClient/gui/PromptDialog.jsx

```diff
@@ -162,6 +162,17 @@ class PromptDialog extends React.Component {
 		return this.styles_;
 	}
 
+	filterOptions(searchKey) {
+		const fullList = this.props.autocomplete;
```

Comment:
Please add a link to your pull request to explain what it does (the animated gif in particular is useful).